### PR TITLE
SIG-2252 Document additional postback advice

### PR DESCRIPTION
**Pull request by Nuss9**
Adds advice not to poll actively for transaction statuses. 
Adds advice on postback implementation process to avoid queue's.

### CODE REVIEW

File: postback.md [hunk 1]
```diff
@@ -4,7 +4,20 @@ title: Postback
...
+
+### Adviced postback implementation flow
+
+To avoid postback queue's for your application, we advice the following flow once a postback arrives at your server:
+  1. Perform checksum validation
+  2. Always return a 200 OK response*
+  3. Save verified postback to storage
+  4. Continue business logic
+
+\* This is security precaution which prevents information on your validation process to return to the - potentially malicious - sender of the postback.
```
Comment (BerendJonkman) [inline postback.md]: Should be: 

> This is **a** security precaution

File: postback.md [hunk 2]
```diff
@@ -4,7 +4,20 @@ title: Postback
...
+Any other response than 2xx will lead to a queue.
```
Comment (BerendJonkman) [inline postback.md]: Not sure if 

> will lead to a queue

 is the right wording. Perhaps it should be:

> will be send to a queue

**Comment by MrJoe** [comment]
fc4161447938f20cb314552cc45ca7e66407be0d LGTM
Please rebase / squash so it can be merged.

**Comment by Nuss9** [comment]
Ready for merge

**Pull request merged**